Vector/EmbeddingGenerator/README.md

```diff
@@ -0,0 +1,175 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+- azure-search
+- azure-cognitive-search
+- azure-openai
+name: Text chunker and embedding skill for Azure Cognitive Search
+description: The custom skill chunks content using the open source LangChain text chunker and then utilizes Azure Open AI service to generate vector embeddings for that content.
+---
+
+# Azure Open AI Embeddings Generator
+
+This custom skill enables generation of vector embeddings for text content which might be created/ingested as part of the Azure Cognitive Search pipeline. This skill requires users to have an [Azure Open AI service](https://learn.microsoft.com/azure/cognitive-services/openai/) provisioned and uses the specified embedding model of Azure Open AI to generate the vector embeddings for the content. Due to token length restrictions on these models, the skill offers a text chunker, which is built on top of [LangChain's text splitter](https://python.langchain.com/en/latest/reference/modules/text_splitter.html).
+
+Details about the various available embeddings model can be found on the [OpenAI concepts page](https://learn.microsoft.com/azure/cognitive-services/openai/concepts/models#embeddings-models). For a versatile model that can work with most search and retrieval applications, it is recommended to utilize the `ada-002` model.
```

Comment:
typo? "embedding models"?

Reply:
Are you talking about the hyperlink? That's an anchor tag on the docs page

Reply:
Misread.